client/src/components/common/ETGeneralLayout/ETGeneralLayout.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <VApp>
+        <VMain class="main">
+            <Header />
+            <div>
+                <slot></slot>
+            </div>
+        </VMain>
+    </VApp>
+</template>
+
+<script>
+import Header from '@/components/eventType/Header.vue';
+
+export default {
+    name: 'ETGeneralLayout',
+    components: {
+        Header
+    }
+};
+</script>
+
+<style lang="scss" scoped>
+.main {
+    background-color: var(--v-background-base);
+    font-family: 'Inter', sans-serif;
```

Comment:
there must be constant variable for Inter font, ask Volodymyr Matievskiy, he must know